src/common/options/osd.yaml.in

```diff
@@ -739,6 +739,14 @@ options:
   default: 32
   flags:
   - startup
+- name: osd_log_slow_op_to_clog
+  type: bool
+  level: advanced
+  desc: Make OSD daemon send every slow op to cluster log
+  fmt_desc: Setting this to false makes osd to not to report every slow op to cluster log
```

Comment:
Phrasing?

Reply:
A native English speaker should probably be consulted, but: I still think the desc should be rephrased.
Also - please reconsider the flag name.

Reply:
Possibly "log_every_slow_op" or "aggregated_slow_ops_logging"? otherwise - it is not clear what are the options selected by the flag.

Reply:
Added more descriptive information for the option `osd_aggregated_slow_ops_logging`